pages/tabs/controllability.page.tsx

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import React, { useState } from 'react';
+import { CodeEditor, CodeEditorProps } from '~components';
```

Comment:
Can we use a different component? Code Editor is kinda special because it needs additional CSP rules, we'd have to update this line here, too: https://github.com/cloudscape-design/components/blob/main/pages/app/index.html#L22.

Reply:
Will do! I'm going to use the container component here